packages/frontend/app/src/session/sessionForm/SessionForm.tsx

```diff
@@ -1,15 +1,16 @@
-import React, { useState, FC } from "react";
-import { put } from "../common/http";
-import { ISession } from "./SessionContainer";
+import React, { useState } from "react";
+
+import { ISession } from "../SessionContainer";
 import { Button, Form, Header } from "semantic-ui-react";
+import * as sessionAPI from "../api/sessionAPI";
 
-interface SessionEditFormProps {
+interface SessionFormProps {
   getSessions: any;
```

Comment:
I know it wasn't you that introduced this but we should also define this as a `Function` as in `Sessions.tsx`